jest.setup.js

```diff
@@ -1,2 +1,2 @@
-process.env.CPV_API_KEY = 'cpv_api_key';
+process.env.CPV_API_KEY = '56c1f1b8-9b5c-41cd-b8f7-872be3500ad3';
```

Comment:
Pelo que eu vi não precisa mais disso já que o teste deu certo

```suggestion
process.env.CPV_API_KEY = 'cpv_api_key';
```